enhancements/ebpf-in-ocp.md

```diff
@@ -0,0 +1,211 @@
+---
+title: ebpf-in-ocp
+authors:
+  - "@danwinship"
+reviewers:
+  - "@TBD, for network observability"
+  - "@TBD, for other existing eBPF use case X"
+  - "@TBD, for other future eBPF use case Y"
+  - "@dave-tucker, for bpfd"
+approvers:
+  - TBD
+api-approvers:
+  - None
+creation-date: 2022-05-25
+last-updated: 2022-05-25
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also:
+replaces:
+superseded-by:
+---
+
+# Guidelines for the Use of eBPF in OCP
+
+## Summary
```

Comment:
@danwinship should add something on listing and addressing support OCP platforms. So that eBPF programs are tested and documented as supported for a known set of OCP architectures (x86,arm ... )

Reply:
@danwinship same with regards to versioning and required kernel levels... so that it will be possible to list dynamically all supported loaded eBPF programs and also address the issue of upgrading etc.